tools/coverage/main.go

```diff
@@ -30,27 +30,24 @@ import (
 )
 
 const (
-	keyCovProfileFileName    = "key-cov-prof.txt"
-	defaultStdoutRedirect    = "stdout.txt"
-	defaultCoverageTargetDir = "./pkg/"
-	defaultGcsBucket         = "knative-prow"
-	defaultPostSubmitJobName = ""
-	defaultCovThreshold      = 80
-)
-
-var (
-	artifactsDir        = "./artifacts/"
-	coverageProfileName = "coverage_profile.txt"
+	keyCovProfileFileName      = "key-cov-prof.txt"
+	defaultStdoutRedirect      = "stdout.txt"
+	defaultCoverageTargetDir   = "."
```

Comment:
Looks like this should be "./". See https://github.com/knative/test-infra/issues/919

Reply:
I also suspected it was the cause when I saw the issue early this week. However, when I compared the logs, there is no difference between the input flags (except the `postSubmitJobName`, which should not be the cause anyway).
[build-log.txt on 06/03](https://00e9e64bacabfb877681d948575f1af41dd1b0e308ca74a345-apidata.googleusercontent.com/download/storage/v1/b/knative-prow/o/logs%2Fci-knative-build-go-coverage%2F1135350603736682497%2Fbuild-log.txt?qk=AD5uMEvzTWMjoAOKp85d_9yEtAaFXTn6kvd7KFndjVLnA_LqnhNE_OwdG4z4mNWQhHJ54A6nNIk3DmVsloKIYrp_vzxjG4oh3bmpCzl5WdGD2mnWmc8tlQzoGne_KgMGM8fwdtVq4m9rgIo7pOz9xJG8QJ63oQePci3EW2vYiWg7OtlafZSalsdy_KjlNJ5Gz4aj_2-gVaP-3rT1llBsakeFrJOH4AkUmGPOsrmT52c8f1JNiWATL1yih0Xfa_a26ad685Dn15XPKDznqd_5SWWRRmLvwCaDRoU1lwiA5KvBYeybs4h-hpZ8CTdF2bMqvo_kzOgc0thN_wcfPa24De_YUB35lVo3g7JnCcTigc3nPt2oxyBuVPj-eEWtWA3_a7ZzxTYXqDMc2xF0Fi8dheAhRC5CiCng5d2FkCLeVKXXKs4b4vQZRiimhJLDlORrY9wNNCoaVZ4wsSRf508IFsYoqTH1ndVg8IBv0T1hh-OExNzQtTRJmv991ImCWV2gcvxbK0pljKqfFodtEqMReHNQaGqk-yjbGlRXvyHUlGfDLm3iTY9oHGJ_hd_sr-cE8O0m9DHk0r59ZVlmT7K6KJ-B4DqS41p47jTeWhsalSkYv7JVfVQIU8DcSMzM82ky1d59RxxlTJt7UfE4sIwfdrfLz-wq7hPzfaXo6x56lOBtLwY5v5RXQKP7T2ViAv55exPn-XaFvQKW8NmJeJx0eSoqinzsZ1MpF6kZm_t-xXWiMX5XEFZV1R-ryrSy_UZjiUG2PZo8_y5YRhdYFCSrtaRRHA78E9dMS06Wlum-VCa4_MTo-jzTfdLRtXaG6LNrnXmrYjjM7N-0YBC3IcHIOdOjpdIfDyWVoQ)
[build-log.txt on 06/04](https://00e9e64bac8339d8eaa8922ec39429c0137103fab404343d33-apidata.googleusercontent.com/download/storage/v1/b/knative-prow/o/logs%2Fci-knative-build-go-coverage%2F1135712993296781312%2Fbuild-log.txt?qk=AD5uMEub79XHYVzZjaECEePxmOJhmTWmBHXUdzpi1-agKehroBUDfOu2eD66-yprvXYdOBCCYx4Xl7yx_QozrO2iCN-Nr0OuiHlnk5z6M-ZUziVPqpu19FpevJRiTT5PyJ7BIYjSdR8hUCMhcZhAARG9xMY_fXKzUz8el-tt3j3CwstyKu2XJn7bXogrQkOxmXs7dTbZCK9n1edub3MqIT1Is1gVj-Z2nviPJGIxRiWOd_YN_j0R60D1FKyMq_jelMADmlBVUL88tuXiESZt_g1j1Py8c1VgbbkfMscqIjtP2mCl8DRUq3PuK2EzbgX2mi2NWjF6PdqFoU9W9ZOroPnkwtAWs7AyId1xYGHJHAuULg2ITxvjB1YIDw_iNqViYsSnlaVDyH6jIyhjUDgNl_XMgO5hCZ0XwM8u2gvSglgviKEmCEJWx8L9kSxX7Qr4pyq2pU-OTYfjACr8OLtizXTKEQWq3H3G2IRGlM1KtdvdE7q1bhX6UatYDDSez3gT_QYfKZ0gFDo_PQNQgUwFBIe9CTGAoTYDs2P44z6y1_DDm0h4I0TDrTUqmFfemwn7YmKCLQs9hY7CrmTHO4VJO9E3bpSQ9q_OADRW8xs3JPjphIw2nGFGj9Oyzl37bDR4rCkIwU0C946nhJNHw3y4vU5ImN-l5yOIfRnYC7kyA__jk9onTA3G8a77Mrt7V5V4-R8h3bT5y5e1tJYDy2k9NvrQX4CxXZBKNWzXtCzK0Tj3uR2zHKMnvYU5FST8Modyl30uSBW_4tKWR5fTdQlN-bjijzCmnrgrldmqMWKZNYgkPGY_9Ksvpy6gVKgoOC7oQ7oujU2P4HIZM7elnd08W_6R8fDsRnnORQ)
I checked with Srinivas and Stephen, and they do not know why either.

I did find there is difference between the `xml` files, but it should not be due to my change here.
[junit_bazel.xml on 06/03](https://00e9e64bacd95fc0a53ac4e149988405efed74beb6a205c9d7-apidata.googleusercontent.com/download/storage/v1/b/knative-prow/o/logs%2Fci-knative-build-go-coverage%2F1135350603736682497%2Fartifacts%2Fjunit_bazel.xml?qk=AD5uMEsAy-lj1r3tG_0UiCPpxFyelSTVRUsN1749_OHkkK2JsuM8EVYQ3GIjxs363HpoOAGE2bPj_ylNe5xliVEZeUmmkKz9idRxLEMKDTky2nt4WFfBxPBpMKwX3_scqsOY1VyZTnvz0zQ8juxK9H_dZfdH1wJ-80fx24gh-6wNYrKvvAYktw9kLRya_Yv3XiI-XKys-N1Ukd_pic5U247KLgbL3DmAZjs3YW8RAoL0Sb2gOT3BggM5dJ-AVIJb7YeJhUL996evmNAk41QEW6FSkIm4K6QkGCBALlSa-ukSf3ysdhTUrHartr7TTZCRF2aCBne6Dhs_yPoCBd_-vTu_kih93LKchBtTWmA92PyqJGf11O44u3RgZQG7VWwIEYqxUuPh2YRyHChtMassKBRdkbNwjCD51LZWciQmT8sK8o6GN6VeJ-1X9j5aO6erQSOYl494a3p5M5r6EyrnkN-qqbyqg0c6HtlrJZSK_gDuGQBQdvI0e1JpDa14IsKWZtzhb4r0kRw607quJlQtIrSEkvNa-GYnEitUb3cXP73d5sAlTDcvlvhN7SfRxB2rK1NyAQxfXSb8ElmUNQHifYTDJ_-YnsYfLgNc-yk5cDLAg8jc1RgafQ2VXKqJyn8uMg4B0cHZnjEcUQiWxO3M7eX7pu6zqgUvhR0-75tNSGTAReayC3Rxfjqp027BOF-NMbm936NpbFjoXN7-IT19OUks5jBFae9WA0HwvIEh9gAa1oAUkTd_jh4LbngRqcWOTjMpQokUwzNwlibRMyuHx9qz4He1Gu_I0QGi22qjds6wSNCTWcgFvriuayGWrwLpJZaMxJMeQlQv-ymz-L3GV4vG85Uc2sovI79F-kqSBaokbType54GLuw)
[junit_bazel.xml on 06/04](https://00e9e64baca698ae0db192c0be1b47aae40082e025299480d7-apidata.googleusercontent.com/download/storage/v1/b/knative-prow/o/logs%2Fci-knative-build-go-coverage%2F1135712993296781312%2Fartifacts%2Fjunit_bazel.xml?qk=AD5uMEuEuXssOLfJWVRbnd2iQenzVcvbC_cNNcMYmnRtM7y61ZBWgCYUGaWae9aDq7Efojlm1rPNm3YPOnvAD8u16f86OepxBIYnfODymLKzQzgS-pLKxxWbY8Jggb7q_7_u4884j7u-bb9SHWJaZszrQ-Eaev6M3Yv35PjoFlvaqlp-1WLLTvcRs6cV3BfJci3Xh5Zwgj8HVt8yrRAeSqRXHkxWLd3z0y00L1rARiW1YyQFZm9O7KuFDhORMgv_BVZwo_JNSuMPza6QbSsx27gojf45xgWVOuoDZM_I-2zUqO3OBjGagV6iAuXKrppUP9WxYDnF_tybsCiMKllonrLCnwVUmdiAeKodLrn7S3mGw--fEJZVi528nXn7yFht-8teAH-eXlGFqLhZ1V-uSfi-5fTP8QsJv6Jbm9S_jAzdTiMGPlE3Dfgrmb3_wax0TtqXw7reBwc-xXABmf_jbc9uR8fU0KC0ZtseW43VrUioEiO4b_b26VRsKDr8ANFUkpy9Zx3HKXpkXDvQ_70ZzyAkR36uaWdCpuC3NR97OV-Toea6Rm5Qt-crwM_Sp29KI_c-HkVVO1UWETysJNArfxyjl0b5QErI6Sdsjz80AEiFhu8QuW1ghFQO0e-n_rAEY_PduI91ae2UyJISKPpx1Hb6N2GEHvvduLMnS8sLz_hH4vbcXFtxbKPM-W6BC_T3PLhizHhpc7kqHQB1Xj5TEBazqGQHpHJwAhZmM67r2fFJJWpmz1LLMcAolqSouZhbpAGbDPUoYtxB2vAl4JMNJaalDQzxx2BdwZri995PGazW2m2enrtKf-jjq4dDP4jNkCAK8iJ1LSB2JBp88iBW-m3IADbqb-NHXIwCP7ABZX8uHxTOABl2NbM)